lib/queries/tile.ts

```diff
@@ -0,0 +1,25 @@
+import { GetTileQuery, ITileQuery } from '../types/index';
+
+/**
+ * @description The default tile query builder
+ */
+export const defaultGetTileQuery: GetTileQuery = ({
+  x,
+  y,
+  z,
+  table,
+  geometry,
+  extent,
+  bufferSize,
+  attributes,
+}: ITileQuery) => `
+SELECT
+  ST_AsMVTGeom(ST_Transform(${geometry}, 3857), TileBBox(${z}, ${x}, ${y}, 3857), ${extent}, ${bufferSize}, false) AS geom,
```

Comment:
perhaps z, x, y should be $1, $2, $3 and passed to the query as values instead of directly as SQL strings in the query

Reply:
That would be an idea or we can make sure x, y and z are numbers. This way we make sure is not SQL injection. I'll make a PR for that.